woocommerce/woocommerce-android
# Creating shipping label: Products cards

Part of: #12281 

### Description
This PR adds the initial screen for creating the shipping label flow. The changes here include:
- The Product card implementation
- The Hazmat card implementation
- The Package card implementation

All the changes in this PR are focused on UI, and the data used in the screens is mocked. On a different PR, I will connect this UI with real data.

### Testing information
1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap con create shipping labels
5. Check that the Product, Hazmat, and Package cards are displayed.
6. Tap on the Product card
7. Check that the product list is expanded
8. Tap again in the Product card
9. Check that the product list is collapsed

### The tests that have been performed
1. Testing that the UI interaction works as expected

### Images/gif

https://github.com/user-attachments/assets/56ab9a08-e3aa-47b5-8007-fb677af4e3dd


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:


p1729175636645019-slack-C02KUCFCSFP
